wallet/lib/pages/detailed_asset_view/widgets/nft_image_asset.dart

```diff
@@ -35,7 +35,7 @@ class NftImageWidget extends StatelessWidget {
                   BlendMode.srcOver,
                 ),
                 image: imageProvider,
-                fit: BoxFit.fitHeight,
+                fit: BoxFit.fitWidth,
```

Comment:
Why we changed this?

Reply:
> Why we changed this?

Hi, As per the issue #924 , We needed to fix the image zoom level so that it does not cut the horizontal sides .